src/Sulu/Component/Rest/ListBuilder/Metadata/FieldDescriptorFactory.php

```diff
@@ -221,6 +222,32 @@ private function getCountFieldDescriptor(AbstractPropertyMetadata $propertyMetad
         );
     }
 
+    private function getCountDistinctFieldDescriptor(AbstractPropertyMetadata $propertyMetadata)
```

Comment:
This function is never called think there is something missing at: https://github.com/sulu/sulu/blob/9834ad41870335a6cdc2cc7c799772e9484b1662/src/Sulu/Component/Rest/ListBuilder/Metadata/FieldDescriptorFactory.php#L88-L93

But based on the above discussion its maybe not needed if we extend the exist DoctrineCountFieldDescriptor